functions/incident_ids.sql

```diff
@@ -0,0 +1,13 @@
+CREATE SEQUENCE IF NOT EXISTS incident_id_sequence INCREMENT 1 START 1;
```

Comment:
@moshloop  @adityathebe 

Should we keep this in `functions`  or `views`

By name, `functions` make more sense, but we already have many functions in the `views` folder

In my mind, the distinction was that generic things go to functions and business logic related things go to `views`

Also, there are tons of functions/triggers in `views`. Should we eventually move them to `functions`  then?

Let me know your thoughts

Reply:
This one in particular cannot be in `views` as the table column refers to it.